frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -162,6 +162,7 @@ function sanitizeFilterParams(filters: Partial<FilterType>): Record<string, any>
     const properties_global = flattenProperties(properties)
 
     return {
+        filters,
```

Comment:
bit dicey! Since this now adds the original `filters` object to everything that was calling sanitizeParams?

And isn't used in `report_user_action`, where the `cohort update` event is triggered from

Reply:
Hmmm I wanted to add this primarily so we can query for insights that are using cohort filters